python/message-service/message_service/__main__.py

```diff
@@ -1,37 +1,39 @@
 import os
-import beeline
-import beeline.propagation.w3c as w3c
-import libhoney
 import random
-from bottle import Bottle, run
-from beeline.middleware.bottle import HoneyWSGIMiddleware
-
-beeline.init(
-    # Get this via https://ui.honeycomb.io/account after signing up for Honeycomb
-    writekey=os.environ.get("HONEYCOMB_API_KEY"),
-    # The name of your app is a good choice to start with
-    api_host=os.environ.get("HONEYCOMB_API_ENDPOINT"),
-    service_name='message-python',
-    debug=True,
-    http_trace_parser_hook=w3c.http_trace_parser_hook,
-    http_trace_propagation_hook=w3c.http_trace_propagation_hook,
-)
 
+from bottle import Bottle, run
+from opentelemetry import trace
+from opentelemetry.exporter.otlp.proto.grpc.trace_exporter import \
```

Comment:
Is the `\` necessary?

Reply:
It makes it to where that import statement can run over to the next line and keep a preferred text width, so I could remove it but then the line would be much longer.  I was following what I saw in other files, fwiw, so I assumed there was a file width preference!

Reply:
Gotcha, makes sense!